.github/ISSUE_TEMPLATE/new-diag-study-template.md

```diff
@@ -0,0 +1,76 @@
+---
+name: New Study Template
+about: Systematically summarize each therapy-focused paper
+title: 'New Paper: [Title]'
+labels: New Paper
+assignees: ''
+
+---
+
+<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
+
+<!--Title: Please edit the title to add the name of the paper after the colon--!>
+
+### Please paste a link to the paper or a citation here:
+
+### What is the paper's [DOI](https://library.uic.edu/help/article/1966/what-is-a-doi-and-how-do-i-use-them-in-citations)?
+
+### Is this paper primarily relevant to Background, Diagnostics, or Therapeutics? (use separate template for diagnostics vs therapeutics)
+
+### Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19
+
+### Questions to answer about each paper:
+
+* **1-2 sentences introducing study, main findings**
```

Comment:
I really love this content, but formatting this as a bulleted list could make it harder for folks to interleave their answers (which is how the issue templates work). I'm going to file a PR into your fork to make some suggested changes. If those look good and you merge them into your branch, they should show up here too! Look for more from me in ~30 mins :)